solc/CommandLineInterface.cpp

```diff
@@ -91,9 +91,10 @@ using namespace solidity::langutil;
 namespace
 {
 
-set<frontend::InputMode> const CompilerInputModes{
+set<frontend::InputMode> const CompilerInputModes {
```

Comment:
```suggestion
set<frontend::InputMode> const CompilerInputModes{
```
It's an initializer list, not function body.